operations/network/customer_accessible_networks/can_to_chn/network/chn_enable.md

```diff
@@ -1,36 +1,37 @@
-- [Migrate phase](#migrate-phase)
-  - [Migrate NCN workers](#migrate-ncn-workers)
-  - [Migrate CSM services (MetalLB)](#migrate-csm-services-metallb)
-  - [Migrate UAN](#migrate-uan)
-  - [Minimizing UAN downtime](#minimizing-uan-downtime)
-    - [Enable CFS for UAN](#enable-cfs-for-uan)
-    - [Notify UAN users](#notify-uan-users)
-  - [Migrate UAI](#migrate-uai)
-  - [Migrate computes (optional)](#migrate-computes-optional)
-    - [Add compute IP addresses to CHN SLS data](#add-compute-ip-addresses-to-chn-sls-data)
-    - [Upload migrated SLS file to SLS service](#upload-migrated-sls-file-to-sls-service)
-    - [Enable CFS layer](#enable-cfs-layer)
-- [Cleanup phase](#cleanup-phase)
-  - [Remove CAN from SLS](#remove-can-from-sls)
-  - [Remove CAN from customizations](#remove-can-from-customizations)
-  - [Remove CAN from BSS](#remove-can-from-bss)
-  - [Remove CAN from CSM services](#remove-can-from-csm-services)
-  - [Remove CAN interfaces from NCNs](#remove-can-interfaces-from-ncns)
-  - [Remove CAN names from NCN hosts files](#remove-can-names-from-ncn-hosts-files)
-- [Update the management network](#update-the-management-network)
-- [Testing](#testing)
+- [Enabling Customer High Speed Network Routing](#enabling-customer-high-speed-network-routing)
+  - [Process overview and warnings](#process-overview-and-warnings)
+  - [Prerequisites](#prerequisites)
+  - [Backup phase](#backup-phase)
+    - [Preparation](#preparation)
+    - [Create system backups](#create-system-backups)
+  - [Update phase](#update-phase)
+    - [Disable CFS for UAN](#disable-cfs-for-uan)
+    - [Update SLS](#update-sls)
+    - [Update customizations](#update-customizations)
+    - [Update CSM service endpoint data (MetalLB)](#update-csm-service-endpoint-data-metallb)
+  - [Migrate phase](#migrate-phase)
+    - [Migrate NCN workers](#migrate-ncn-workers)
+    - [Migrate CSM services (MetalLB)](#migrate-csm-services-metallb)
+    - [Migrate UAN](#migrate-uan)
+    - [Minimizing UAN downtime](#minimizing-uan-downtime)
+      - [Enable CFS for UAN](#enable-cfs-for-uan)
+      - [Notify UAN users](#notify-uan-users)
+    - [Migrate UAI](#migrate-uai)
+    - [Migrate computes (optional)](#migrate-computes-optional)
+      - [Add compute IP addresses to CHN SLS data](#add-compute-ip-addresses-to-chn-sls-data)
+      - [Upload migrated SLS file to SLS service](#upload-migrated-sls-file-to-sls-service)
+      - [Enable CFS layer](#enable-cfs-layer)
+  - [Cleanup phase](#cleanup-phase)
+    - [Remove CAN from SLS](#remove-can-from-sls)
+    - [Remove CAN from customizations](#remove-can-from-customizations)
+    - [Remove CAN from BSS](#remove-can-from-bss)
+    - [Remove CAN from CSM services](#remove-can-from-csm-services)
+    - [Remove CAN interfaces from NCNs](#remove-can-interfaces-from-ncns)
+    - [Remove CAN names from NCN hosts files](#remove-can-names-from-ncn-hosts-files)
+  - [Update the management network](#update-the-management-network)
+  - [Testing](#testing)
```

Comment:
No need to include the page itself in its own TOC. I know some editors do that by default, alas.

```suggestion
- [Process overview and warnings](#process-overview-and-warnings)
- [Prerequisites](#prerequisites)
- [Backup phase](#backup-phase)
  - [Preparation](#preparation)
  - [Create system backups](#create-system-backups)
- [Update phase](#update-phase)
  - [Disable CFS for UAN](#disable-cfs-for-uan)
  - [Update SLS](#update-sls)
  - [Update customizations](#update-customizations)
  - [Update CSM service endpoint data (MetalLB)](#update-csm-service-endpoint-data-metallb)
- [Migrate phase](#migrate-phase)
  - [Migrate NCN workers](#migrate-ncn-workers)
  - [Migrate CSM services (MetalLB)](#migrate-csm-services-metallb)
  - [Migrate UAN](#migrate-uan)
  - [Minimizing UAN downtime](#minimizing-uan-downtime)
    - [Enable CFS for UAN](#enable-cfs-for-uan)
    - [Notify UAN users](#notify-uan-users)
  - [Migrate UAI](#migrate-uai)
  - [Migrate computes (optional)](#migrate-computes-optional)
    - [Add compute IP addresses to CHN SLS data](#add-compute-ip-addresses-to-chn-sls-data)
    - [Upload migrated SLS file to SLS service](#upload-migrated-sls-file-to-sls-service)
    - [Enable CFS layer](#enable-cfs-layer)
- [Cleanup phase](#cleanup-phase)
  - [Remove CAN from SLS](#remove-can-from-sls)
  - [Remove CAN from customizations](#remove-can-from-customizations)
  - [Remove CAN from BSS](#remove-can-from-bss)
  - [Remove CAN from CSM services](#remove-can-from-csm-services)
  - [Remove CAN interfaces from NCNs](#remove-can-interfaces-from-ncns)
  - [Remove CAN names from NCN hosts files](#remove-can-names-from-ncn-hosts-files)
- [Update the management network](#update-the-management-network)
- [Testing](#testing)
```

Reply:
We avoid including the page itself in its own TOC, since it does not add any information, just clutter. I know that some editors put it in there, but that is mainly for cases where a document has multiple top-level headers. However, in this repository we do not allow multiple top-level headers, so for this repository it should be omitted.

Reply:
Maybe I need to reload again. Let me check. Sometimes github likes to torment me.

Reply:
I concur -- I no longer see the issue. My bad. Marking this resolved.